### ISSUE TITLE
Question about a license

### ISSUE THREAD

**Issue by xjack03** [anagram4wander/VirtualizingObservableCollection#9]
Hello anagram4wander,

would it be possible to add an open-source license, best if it be https://opensource.org/licenses/MIT.

I would like to extend this project and include it in my open-source SW.

Thanks, JP

**Comment by LeeHoward1** [comment]
Hi anagram4wander,

I too would like to know what license your code is released under. Specifically, I would need the license explicitly stated somewhere. A license file in the root directory is a typical way to do this. Please let me know if you don't intend to add a license, since I won't be able to use your code in that case.

Thanks,

Lee

**Comment by anagram4wander** [comment]
Open license - feel free to use it.

**Comment by LeeHoward1** [comment]
Thanks for the quick reply. I work for a company that requires any code we use to have a specified license. Would you consider applying one of the available licenses? Perhaps the MIT license, which essentially captures the "Open license" features?

**Comment by anagram4wander** [comment]
The mit license is fine.. I will update when I get a chance...

**Comment by LeeHoward1** [comment]
Okay, thank you!

**Comment by WilhelmJP** [comment]
Hello anagram4wander,

for the same reasons as Lee, I am also very interested in getting your lib in connection with the proposed MIT license. Could you please tell me if this is not possible in the near future?

Thanks, 
Wilhelm

**Comment by sjb-sjb** [comment]
I agree with the comments on licensing, which I see have been open for a year or so. We need a license posted on the GitHub page. It may be great software but without a posted license it is not usable.

**Comment by michael-fraser-epm** [comment]
The license file was never committed to the Main branch.  Can someone please add the MIT license to the main branch?
